### PR TITLE
add: cbr support

### PR DESCRIPTION
**Pull request by Aayush-Abhyarthi**
### Description

This PR adds cbr support in App Configuration module

### Release required?

- [ ] No release
- [ ] Patch release (`x.x.X`)
- [X] Minor release (`x.X.x`)
- [ ] Major release (`X.x.x`)

##### Release notes content

This PR adds the cbr support in the App Configuration module.

### Run the pipeline

If the CI pipeline doesn't run when you create the PR, the PR requires a user with GitHub collaborators access to run the pipeline.

Run the CI pipeline when the PR is ready for review and you expect tests to pass. Add a comment to the PR with the following text:

```
/run pipeline
```

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### For mergers

- Use a conventional commit message to set the release level. Follow the [guidelines](https://terraform-ibm-modules.github.io/documentation/#/merging.md).
- Include information that users need to know about the PR in the commit message. The commit message becomes part of the GitHub release notes.
- Use the **Squash and merge** option.

### CODE REVIEW

File: examples/complete/main.tf [hunk 1]
```diff
@@ -14,6 +14,36 @@ module "resource_group" {
...
+  name           = "${var.prefix}-vpc"
+  resource_group = module.resource_group.resource_group_id
+  tags           = var.resource_tags
+}
+
+##############################################################################
+# Create CBR Zone
+##############################################################################
+module "cbr_zone" {
+  source           = "git::https://github.com/terraform-ibm-modules/terraform-ibm-cbr//cbr-zone-module?ref=v1.2.0"
```
Comment (MatthewLemmond) [inline examples/complete/main.tf:37]: Should use the terraform registry source/version such as:
```
source = terraform-ibm-modules/cbr/ibm//modules/cbr-zone-module
version = 1.27.0
```

File: examples/basic/version.tf [hunk 2]
```diff
@@ -6,7 +6,7 @@ terraform {
   required_providers {
     ibm = {
       source  = "IBM-Cloud/ibm"
-      version = "1.49.0"
+      version = "1.65.0"
```
Comment (MatthewLemmond) [inline examples/basic/version.tf:9]: The version should also be updated in the other example and the root version.tf

**Review by MatthewLemmond — CHANGES_REQUESTED** [review]

File: examples/complete/main.tf (hunk 1 above)
Comment (Aayush-Abhyarthi) [reply examples/complete/main.tf]: changed

File: examples/basic/version.tf (hunk 2 above)
Comment (Aayush-Abhyarthi) [reply examples/basic/version.tf:9]: Updated

**Review by MatthewLemmond — APPROVED** [review]

**Pull request merged**